utils/tests/online.py

```diff
@@ -128,7 +128,7 @@ def test_purge_old(self):
         self.redis_conn.set('default_counter:2015-10', 'stuff2')
 
         tc.purge_old()
-        self.assertEqual(['default_counter:2015-10'],
+        self.assertEqual([b'default_counter:2015-10'],
```

Comment:
@madisonb thoughts on this? when you say sanitize with six.binary_type() what do you mean exactly?